public/docs/ts/_cache/tutorial/toh-pt6.jade

```diff
@@ -115,15 +126,24 @@ block dont-be-distracted-by-backend-subst
 
 :marked
   We returned a !{_Promise} resolved with mock heroes.
-  It may have seemed like overkill at the time, but we were anticipating the
+  It may have seemed like an overkill at the time, but we were anticipating the
```

Comment:
Yeah, when I saw it, I realized that "an" would make sense, but for some reason it's never used that way.